parser_library/src/processing/statement_analyzers/hit_count_analyzer.cpp

```diff
@@ -106,30 +119,25 @@ hit_count_analyzer::statement_type hit_count_analyzer::get_stmt_type(const conte
         return true;
     };
 
-    static constexpr auto range_adj = [](range r) {
-        r.end.column = 72;
-        return r;
-    };
-
     switch (m_stmt_type)
     {
         case hlasm_plugin::parser_library::processing::hit_count_analyzer::statement_type::REGULAR:
             if (macro_init_incrementer(res_stmt))
             {
                 m_stmt_type = statement_type::MACRO_INIT;
-                m_last_macro_init_r = res_stmt->stmt_range_ref();
+                m_last_macro_init_line_ranges = stmt_lines_compacter(res_stmt->stmt_range_ref());
```

Comment:
Please add a TODO somewhere (maybe next to m_last_macro_init_line_ranges, or somewhere in the macro processor?) that we should probably mark the special macro statements (macro, mend, prototype) somehow, so that we don't have to keep recreating this state machine.

Reply:
Created `TODO`